cocos/3d/CCMesh.h

```diff
@@ -94,10 +95,39 @@ class CC_DLL Mesh : public Ref
     /**get per vertex size in bytes*/
     int getVertexSizeInBytes() const;
 
-    /**texture getter and setter*/
+    /**
+     * set diffuse texture
+     * @param texPath texture path
+     */
     void setTexture(const std::string& texPath);
+    /**
+     * set diffuse texture
+     * @param tex texture to be set
+     */
     void setTexture(Texture2D* tex);
```

Comment:
keep the old interface.
This interface for person who does not understand material is enough.